cms/djangoapps/contentstore/helpers.py

```diff
@@ -345,6 +345,14 @@ def _import_xml_node_to_parent(
     else:
         for child_node in child_nodes:
             _import_xml_node_to_parent(child_node, new_xblock, store, user_id=user_id)
+
+    # ToDo: Check the better place to call it
+    # I tried to call it in the xml_block.py in the parse_xml() function,
+    # but the usage_key is not persited yet there
```

Comment:
nit: could you update or remove this comment?

a) it says "ToDo", but we don't have a plan to address
b) It's not correct: by the end of `xml_block.parse_xml`, the returned xblock has a location / usage key applied.

Reply:
> b) It's not correct: by the end of xml_block.parse_xml, the returned xblock has a location / usage key applied.

What I mean is that at the end of `xml_block.parse_xml`, it has a usage_key in the form of `'block-v1:SampleTaxonomyOrg1+STC1+2023_1+type@html+block@localid_128844752808688'` (with localid). From what I understood, the key is persisted later and become: `'block-v1:SampleTaxonomyOrg1+STC1+2023_1+type@vertical+block@aaf8b8eb86b54281aeeab12499d2cb0b'` (the one that we want to apply the tags to).
This comment was to discuss it in the review, but I forgot to mark it here. Sorry!
Anyway, I removed it here: 2eb02303b17725cf2f5908b6b20658e0711d6d47

Reply:
Huh interesting about the "localid".. I don't know what that's for.
But thank you for removing the todo :)